tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/external/iso_3166_status.txt

```diff
@@ -100,7 +100,7 @@ GA	;	Officially assigned	;	Gabon (le)
 GB	;	Officially assigned	;	United Kingdom of Great Britain and Northern Ireland (the)
 GC	;	Indeterminately reserved	;	Refers to the Patent Office of the Cooperation Council for the Arab States of the Gulf (GCC) and reserved at the request of World Intellectual Property Organization (WIPO) for use in Standard ST.3.
 GD	;	Officially assigned	;	Grenada
-GE	;	Officially assigned	;	Georgia
+GE	;	Formerly used	;	Georgia
```

Comment:
Now this seems odd..  Formerly Used??

Reply:
"The code GE was formerly used for the Gilbert and Ellice Islands (GE, GEL, --) and has been reused and reassigned to Georgia. See also code element GEKI."

I guess like SK, "Formerly used" for GE is a mistake on my part, misunderstood instructions...

Reply:
reverted